flake.nix

```diff
@@ -1,6 +1,7 @@
 {
   description = "plutarch";
 
+  inputs.nixpkgs-2111 = { url = "github:NixOS/nixpkgs/nixpkgs-21.11-darwin"; };
```

Comment:
I think we want to use `follows` here.

Reply:
I think I'll just remove the change completely -- this PR shouldn't be concerned with it anyway. Especially since it's only a bug in Nix.